Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkCosmosClusterManager.java

```diff
@@ -242,16 +242,10 @@ public ClusterContainer refresh() {
                                 .flatMap(accountsResp -> Observable.from(accountsResp.items()))
                                 .map(accountBasic -> Pair.of(subAccountsObPair.getLeft(), accountBasic)))
                 .flatMap(subAccountBasicPair -> {
-                    // accountBasic.id is the account detail absolute URI path
-                    URI accountDetailUri = getResourceManagerEndpoint().resolve(subAccountBasicPair.getRight().id());
```

Comment:
I prefer not touch the legacy codes here.

Reply:
Here the change is, I put getting ADLA account details code in a seperate method `getAzureDataLakeAccountDetail`. If we don't make this change here, there will be duplicate codes in method `AzureSparkCosmosClusterManager::getAzureDataLakeAccountDetail` and `SynapseCosmosSparkPool::getConfigurationInfo`.